src/TestFramework/TestFramework/Attributes/DataSource/TestIdGenerationStrategyAttribute.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.VisualStudio.TestTools.UnitTesting;
+
+/// <summary>
+/// Specifies how to generate test ID.
+/// </summary>
+[AttributeUsage(AttributeTargets.Assembly, AllowMultiple = false)]
+public class TestIdGenerationStrategyAttribute : Attribute
+{
+    /// <summary>
```

Comment:
Should we add a `DefaultStrategy` const here?

Reply:
What would it bring for you?

Reply:
Good question, probably nothing - it might let user to know what is the current default.

Reply:
Will keep as-is for now then. I expect that the comments are providing enough information and I don't expect people to use explicitly `FullyQualified`.